buf.gen.yaml

```diff
@@ -1,15 +1,20 @@
-version: v1
```

Comment:
Were there any manual changes or was this entire diff from running `buf config migrate`?

Reply:
No manual changes. The entire diff is from `buf config migrate`.